src/commands.ts

```diff
@@ -248,6 +248,147 @@ export async function createNewProject(ctx: WorkspaceContext): Promise<void> {
     }
 }
 
+/**
+ * Prompts the user to input project details and then executes `swift package init`
+ * to create the project.
+ */
+export async function browsePackageIndex(ctx: WorkspaceContext): Promise<void> {
+    // The context key `swift.addPackageRefactoringAvailable` only works if the extension has been
+    // activated. As such, we also have to allow this command to run when no workspace is
+    // active. Show an error to the user if the command is unavailable.
+    if (!ctx.toolchain.swiftVersion.isGreaterThanOrEqual(new Version(6, 0, 0))) {
+        vscode.window.showErrorMessage(
+            "Browsing the package index is only available starting in swift version 6.0.0."
+        );
+        return;
+    }
+
+    const panel = vscode.window.createWebviewPanel(
```

Comment:
Can we break out some of the web view stuff, into a general function that can be used by other systems. Currently when you right click on a package dependency and select `View Repository` it opens a web page in your default browser. Would be cool if we could do that inside VSCode instead in a similar manner to how this works.